evm/test/NttManager.t.sol

```diff
@@ -226,6 +227,27 @@ contract TestNttManager is Test, INttManagerEvents, IRateLimiterEvents {
         nttManager.setTransceiver(address(0x123));
     }
 
+    function test_maxOutTransceivers() public {
+        // Let's register a transceiver and then disable it
+        DummyTransceiver e = new DummyTransceiver(address(nttManager));
+        nttManager.setTransceiver(address(e));
+        nttManager.removeTransceiver(address(e));
+
+        // We should be able to register 64 transceivers total
+        for (uint256 i = 0; i < 63; ++i) {
```

Comment:
Is it straightforward to assert on the length of the transceivers in Foundry? it could be nice to add that it actually is equal to 64 in addition to reverting

Reply:
It's 63 in the loop as we've already added a transceiver before the loop :)

But yeah I'll check to see if there's a view for us to assert there are actually 64 transceivers and make that change

Reply:
Sure sounds good. I was only suggesting adding if it was easy. Not worth blocking on